server/routes/hello.js

```diff
@@ -52,7 +52,8 @@ module.exports = {
               return reply(getUserDbErr + 'error getting user from database');
             }
             request.cookieAuth.set({ id: userId, });
-            reply.redirect('/');
+            reply.redirect('/').
+            state('new-user', fbUserId.toString());
```

Comment:
this seems weird, are you sure you do not want to set this in the session you have aready created (see `request.cookieAuth.set({ id: userId, });` two lines above).

Also indentation